### DISCUSSION TITLE
Secrets module ?

### DISCUSSION THREAD

**Discussion by giannik** [OrchardCMS/OrchardCore#16935] [General]
Hello I remember there was a pr related to a secrets module  for maintaining global wide secrets . But I can't seem to find it anymore. Does anyone know what happened to that PR?

**Comment by hishamco** [comment]
I think it's related to #6421

**Comment by Piedone** [comment]
I suppose you're referring to https://github.com/OrchardCMS/OrchardCore/pull/7891.

**Reply by hishamco** [reply]
Right

**Comment by giannik** [comment]
so i guess its out of scope.

**Reply by Piedone** [reply]
Not really something that anybody would be actively working on right now, at least.
